KTLOSNaxxLoot.json

```diff
@@ -78,7 +83,7 @@
 			"wowID":"22370"
 		},
 		"Desecrated Waistguard":{
-			"prio":["Snydicate","Tanks","LC","EP/GP"],
+			"prio":["Chart(1)","Supertoxic(1)","Johnscar(2)","Killstep(2)","Rogues","LC","EP/GP"],
```

Comment:
Consider removing this for now and leaving it as Tank 4pc first.
That or make it clear that tanks are getting their 4pc first before this will go out to any rogues.
Here and else where on the tier.

Reply:
Please send me the list of people in raid 1 and raid 2 like I asked for.

Reply:
Raid 1: Snydi, Aisli, Tuglight, Dill, Healslol, Vajeen, Impactt
Raid 2: Frau, Wizzo, Beeny

Reply:
are those complete or partial? Won't we need at least 4+ in raid 2? Also should killstepp be on the raid 1 list?

Reply:
Raid 1 is complete, we can swap Killstepp / Impactt but I don't think it makes a difference.
Raid 2 is partial, but we actually don't have reliable proven tanks (IMO) aside from the 3 listed.

Reply:
Give Killstepp a set in raid 1 and swap one of the other warriors to raid 2?

Reply:
I'd rather not, since R2 needs 4-5+ tank capable warriors so swapping in 1 r1 warrior doesn't fix anything.

Reply:
one of them, Rozzy and Chrixus seems like a start?

Reply:
@dillmcpickle

Reply:
R1: Snydi, tug, aisli, Dill, Mulligen, Vajeen, Healslol, Killstepp. can also drop one of the last 2 for PQ if were doing druids
R2: Frau, Beeny, Wizzo, Chrixus, Rozzy, Quigbert, Charts druid?, ehh builtlike?